docs/installation.rst

```diff
@@ -24,14 +24,24 @@ There are different ways to install |project_name|:
 Distribution Package
 --------------------
 
-Some Linux and BSD distributions might offer a ready-to-use ``borgbackup``
+Some distributions might offer a ready-to-use ``borgbackup``
 package which can be installed with the package manager.  As |project_name| is
 still a young project, such a package might be not available for your system
 yet.
 
-* On **Arch Linux**, there is a package available in the AUR_.
+============ ===================== =======
+Distribution Source                Command
+============ ===================== =======
+Arch Linux   AUR_
+Debian       `unstable/sid`_       ``apt install borgbackup``
+Ubuntu       `Xenial Xerus 15.04`_ ``apt install borgbackup``
```

Comment:
well, some OSX users may not be familiar with OSX, for example. it also makes it clear what the package name is: some may (mistakenly) use "borg" or "borg-backup" for example.

as for `apt-get install`, `apt install` now works starting from jessie. i assumed the same on Ubuntu.

will correct xerus.